conda/plugins/post_solves/signature_verification.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""Register signature verification as a post-solve plugin."""
+from .. import CondaPostSolve, hookimpl
+
+
+@hookimpl
+def conda_post_solves():
+    from ...trust.signature_verification import signature_verification
+
+    yield CondaPostSolve(
+        name="signature-verification",
+        action=signature_verification,
+    )
```

Comment:
So, maybe crazy idea, should this move into conda-content-trust? (including the trust module)

Reply:
We've discussed this, and want to merge this first, then refactor.